### DISCUSSION TITLE
Suggestions for improving the chia2.0 interface

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/chia-blockchain-gui#2382] [Ideas]
The upper and lower slider in the left sidebar under the low-size window is too unsight. it is suggested to refer to the following content for improvement.
![微信截图_20230814120206](https://github.com/Chia-Network/chia-blockchain/assets/97425684/e4d00565-6bcd-4eda-8c92-c4e0f74f001b)
![微信截图_20230814120220](https://github.com/Chia-Network/chia-blockchain/assets/97425684/bff61568-f0ff-437b-be97-3d946fdb1f8a)
![微信截图_20230814120234](https://github.com/Chia-Network/chia-blockchain/assets/97425684/2cb321dd-df94-4eb3-8980-22879f071773)

**Comment by NTOHPN** [comment]
![image](https://github.com/Chia-Network/chia-blockchain/assets/97425684/4c1da2ba-71db-425a-9df5-81d967b0ea04)

**Comment by BrandtH22** [comment]
Hey Ntohpn, thank you for your request. 
I have passed it along to our UI design team, feel free to add any related additional information to this discussion :+1:

**Reply by NTOHPN** [reply]
Thank you.

**Comment by BrandtH22** [comment]
@Ntohpn  , note I transferred this enhancement request to the GUI repo for better visibility 👍
